server/handlers.go

```diff
@@ -1005,3 +1005,12 @@ func (s *Server) tokenErrHelper(w http.ResponseWriter, typ string, description s
 		s.logger.Errorf("token error response: %v", err)
 	}
 }
+
+// Check for username attribute override from connector. Defaults to "Username".
+func usernameAttr(conn connector.PasswordConnector) string {
+	attr := "Username"
+	if conn, ok := conn.(connector.UsernameAttributeProvider); ok {
+		attr = conn.UsernameAttribute()
```

Comment:
If the connector returns `""` we should default to `Username`. e.g. the LDAP connector can just return `""` instead of setting a local default.